src/System.Private.Uri/src/System/UriBuilder.cs

```diff
@@ -114,14 +114,10 @@ string extraValue
             {
                 Extra = extraValue;
             }
-            catch (Exception exception)
+            catch (ArgumentException)
```

Comment:
This does not appear to be a valid change.  Previously it was catching all exceptions (exception OOMs) and wrapping them in an ArgumentException... now you're only catching ArgumentException.  There are theoretically other exceptions that could emerge from set_Extra.  Please revert this.